linalg/operator.hpp

```diff
@@ -415,6 +415,11 @@ class IdentityOperator : public Operator
    virtual void MultTranspose(const Vector &x, Vector &y) const { y = x; }
 };
 
+/// Returns true if the P operator identity.
```

Comment:
```suggestion
/// Returns true if the P operator is identity.
```
A comment is also warranted here saying that historically `NULL` had been used to signify the identity operator as an optimization.